applications/CoSimulationApplication/python_scripts/coupling_operations/compute_boundary_force.py

```diff
@@ -0,0 +1,163 @@
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+# Importing the base class
+from KratosMultiphysics.CoSimulationApplication.base_classes.co_simulation_coupling_operation import CoSimulationCouplingOperation
+
+# Additional imports
+from KratosMultiphysics.time_based_ascii_file_writer_utility import TimeBasedAsciiFileWriterUtility
+
+# CoSimulation imports
+import KratosMultiphysics.CoSimulationApplication.co_simulation_tools as cs_tools
+
+def Create(*args):
+    return ComputeBoundaryForce(*args)
+
+class ComputeBoundaryForce(CoSimulationCouplingOperation):
+    """This operation is used to compute forces in a boundary, based on the pressure.
+    TODO:
+    - add messages with different echo-levels
+    - add tests
+    - more cleanup
+    """
+    def __init__(self, settings, solver_wrappers, process_info):
+        super().__init__(settings, process_info)
+        self.model = solver_wrappers[self.settings["solver"].GetString()].model
+        self.model_part_name = self.settings["model_part_name"].GetString()
+        self.model_part = self.model[self.model_part_name]
+        self.write_output_file = self.settings['write_output_file'].GetBool()
+        self.format = self.settings["print_format"].GetString()
+
+        self.width = self.settings["width"].GetDouble()
+        # If 2D case: width from parameters is used
+        # If 3D case: width is not used
+        domain_size = self.model_part.ProcessInfo[KM.DOMAIN_SIZE]
+        if domain_size == 3:
+            self.width = 1
+        
+        # Detect 'End' as a tag and replace it by a large number
+        if(self.settings.Has('interval')):
+            if(self.settings['interval'][1].IsString()):
+                if(self.settings['interval'][1].GetString() == 'End' or self.settings['interval'][1].GetString() == 'end'):
+                    self.settings['interval'][1].SetDouble(1e30)
+                else:
+                    raise Exception('The second value of interval can be \'End\' or a number, interval currently:' + self.settings['interval'].PrettyPrintJsonString())
+        self.interval = self.settings["interval"].GetVector()
```

Comment:
please use the `IntervalUtility` for this

Reply:
Thanks for this hint, it reduced all of this to a single line.